### DISCUSSION TITLE
Using multiple tables in Form view [Dec 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4712] [Discord Threads]
> 🙋‍♂️ **zblesk**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Form view are specific to a table currently  
#  
> 🙋‍♂️ **zblesk**  
#### Thanks.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1043988985388728493)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
